src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  @for $i from 1 through 5 {
+    &--#{$i} :nth-child(-n + #{$i}) {
+      background-image: url(../images/star-active.svg);
+      background-repeat: no-repeat;
```

Comment:
You shouldn't repeat properties background repeat and background position cause you already have the properties above